src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -172,18 +173,27 @@ Amplify.configure({
 ```
```

Comment:
Is there anything more we want to add to the Recap above to illustrate a tie-in to real-world use or application?